### PR TITLE
DCS-363 Adding database refresh script

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: database_refresh/01-configmap-refresh-script.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          echo "Not running refresh this week, refreshes only happens on odd week numbers, check back next week."
+          exit 0
+        fi
+        echo "${DB_HOST}:5432:${DB_NAME}:${DB_USER}:${DB_PASS}" > ~/.pgpass
+        echo "${DB_HOST_PREPROD}:5432:${DB_NAME_PREPROD}:${DB_USER_PREPROD}:${DB_PASS_PREPROD}" >> ~/.pgpass
+        chmod 0600 ~/.pgpass
+        set -x
+        pg_dump -h $DB_HOST -U $DB_USER -Fc --no-privileges -v --file=/tmp/db.dump $DB_NAME
+        pg_restore -h $DB_HOST_PREPROD -U $DB_USER_PREPROD --clean --no-owner -v -d $DB_NAME_PREPROD /tmp/db.dump
+        #psql -h $DB_HOST -U $DB_USER $DB_NAME
```
Comment (brightonsbox) [inline database_refresh/01-configmap-refresh-script.yaml]: Did you want these commented out?

File: database_refresh/02-cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+    apiVersion: batch/v1beta1
+    kind: CronJob
+    metadata:
+      name: db-refresh-job
+    spec:
+      schedule: "30 10 * * 0"
```
Comment (brightonsbox) [inline database_refresh/02-cronjob.yaml]: 10:30 on a Sunday

Comment (andrewrlee) [reply database_refresh/02-cronjob.yaml:7]: err. yes... that's the same as the other jobs

File: database_refresh/01-configmap-refresh-script.yaml (hunk 1 above)
Comment (andrewrlee) [reply database_refresh/01-configmap-refresh-script.yaml]: removed these and added missing adhoc file

File: database_refresh/02-cronjob.yaml (hunk 2 above)
Comment (brightonsbox) [reply database_refresh/02-cronjob.yaml:7]: Sorry, wasn't anything wrong with the cron, I was just echoing it back when I'd parsed it. Ignore me! 🙈

**Pull request merged**